rules/utils/get-docs-url.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const path = require('path');
+
+const repoUrl = 'https://github.com/sindresorhus/eslint-plugin-unicorn';
+
+module.exports = (ruleName, commitHash) => {
```

Comment:
Not sure about the commit hash as we don't use it currently.

Reply:
It's used in a few other plugins that I sent PRs to, so I figured I would leave it in here. I can remove it if you want though.

Reply:
I think we should remove it as long as we don't need it. It adds "complexity" and should be maintained later on if perhaps the function gets more complex. But that's up to Sindre to decide.